testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -49,6 +51,26 @@ class BaseITP(ParserBase):
                       'resids', 'resnames',
                       'segids', 'moltypes', 'molnums',
                       'bonds', 'angles', 'dihedrals', 'impropers']
+
+    guessed_attrs = ['elements', ]
+
+    @pytest.fixture
+    def guessed_types(self, top):
+        return top.types.values
+
+    @pytest.fixture
+    def guessed_masses(self, top):
+        masses = np.array(top.masses.values)
+        names = np.array(top.names.values)
+
+        empty = []
+        for a in masses:
+            empty.append(np.isnan(a))
```

Comment:
```suggestion
        empty = np.isnan(masses)
```

`np.isnan` should return an array of values if given an array.

Although see my comments above about potentially rewriting tests to just use `top` as the fixture instead of `guessed_masses`.